ERCS/erc-chain-identifiers.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: 7785
+title: On-chain registration of chain identifiers
+description: Derive chain identifiers by hashing their chain name and use ENS to map chain names to these identifiers
+author: Marco Stronati (@paracetamolo), Jeff Lau <jeff-lau@live.com>
+discussions-to: https://ethereum-magicians.org/t/on-chain-registration-of-chain-identifiers/21299
+status: Draft
+type: Meta
```

Comment:
```suggestion
type: ERC
```

Reply:
I updated this and pushed a new commit - thanks

Reply:
@abcoathup - it lookd like type: ERC is not a valid entry - ok to go back to Meta? 
<img width="1020" alt="image" src="https://github.com/user-attachments/assets/a6a9c110-1989-42bb-a8d4-b8b270ba8a9f">

Reply:
Use Standards Track